jsons/parameters.json

```diff
@@ -3537,6 +3570,7 @@
     "default_value": "trypsin",
     "description": "Enzyme: Rule of protein cleavagePossible cleavages are :\n    argc           -> [R]|{P}\n    aspn           -> [X]|[D]\n    aspn_gluc\n    chymotrypsin   -> [FMWY]|{P}\n    chymotrypsin_p -> [FMWY]|[X]\n    cnbr           -> [M]|{P}\n    elastase       -> [AGILV]|{P}\n    formic_acid    -> [D]|{P}\n    gluc\n    lysc\n    lysc_p\n    lysn\n    no_cleavage\n    nonspecific\n    pepsina\n    semi_chymotrypsin\n    semi_gluc\n    semi_tryptic\n    thermolysin_p\n    top_down\n    trypsin\n    trypsin_chymotrypsin\n    trypsin_cnbr\n    trypsin_p\n    lysc_gluc",
     "key_translations": {
+      "comet_style_1": "search_enzyme_number",
```

Comment:
Okay, I was not aware that the enzyme param could be a list? 

From the description of the param and the way how it is implemented and translated atm it is always a string with a single enzyme name. Or was that an idea that it could be a list, which is not yet implemented? ^^ If this was supposed to always be a list, I would hence also refactor the single enzyme to a 1-element list, cause I find it inconsistent otherwise.

Reply:
No, it's not a list, but if you look at the entries, you can see multiple enzymes specified like this: `lysc_gluc`